guide/16-geoenrichment-guides/part5_generate_reports.ipynb

```diff
@@ -0,0 +1,980 @@
+{
```

Comment:
<p>the code snippet does not use <code>laborforce_profile</code>  instead it uses <code>business_loc</code>.  One of this has to be updated</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/755/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>